src/types.ts

```diff
@@ -64,6 +68,9 @@ export interface MarketplaceCreator {
   creatorAddress: string
   storeConfigAddress: string
   preview?: Nft[]
+  twitterHandle?: string
+  profile?: TwitterProfile | null
```

Comment:
A marketpalce doesn't have a twitterHandle or profile. Can you drop these to avoid confusion?